src/main.rs

```diff
@@ -78,3 +79,10 @@ fn version() -> &'static str {
     // FIXME when we have non-nightly channels, we shouldn't hardwire the "nightly" string here.
     concat!(env!("CARGO_PKG_VERSION"), "-nightly", include_str!(concat!(env!("OUT_DIR"), "/commit-info.txt")))
 }
+fn help() -> &'static str {
+    r#" 
+    --version or -V to print the version and commit info
+    --help or -h for this message
+    Other input starts the RLS as a language server 
```

Comment:
Ah, this is not quite right. If there are no options provided, then we start as a language server. If you pass any other argument, we start in command line mode.